.travis.yml

```diff
@@ -1,10 +1,12 @@
 dist: bionic
+
 addons:
   chrome: stable
-language: ruby
 
-notifications:
```

Comment:
This is dead configuration, overridden by lines 34-37 below, and has been cruft for 7 years.